### ISSUE TITLE
Key presses not registering/Multiple presses needed

### ISSUE THREAD

**Issue by Kausta** [hellerve/e#15]
On WSL, e usually doesn't respond to key presses on first click. It is more apparent on ESC to exit. It is either WSL related if it only occurs on it, otherwise Linux related.

**Comment by hellerve** [comment]
Okay, I’ll probably have to do some testing to figure out whats happening there. Does it happen when typing text as well?

**Comment by Kausta** [comment]
No, typing is instant and working correctly. It only occurs with commands.

**Comment by hellerve** [comment]
Okay, I’ll try and work on it tonight. Will report back when I’ve fixed it.

**Comment by ghost** [comment]
I can't reproduce this issue on my end.

I've tried it on Ubuntu 16.04 and Fedora 26, I will give it a try later on some other systems just to see if occurs anywhere else.

**Comment by Kausta** [comment]
I also can't reproduce on Ubuntu 17.10 or Arch Linux. It only happens in Windows Subsystem for Linux so it is probably WSL specific (or Windows specific in general but that can't be known yet).
